src/Car.php

```diff
@@ -2,20 +2,30 @@
 
 namespace CarStore;
 
+require_once 'vendor/autoload.php';
+require_once 'src/utils.php';
+
+use CarStore\CarModel;
+
+
 readonly class Car
 {
     public string $name;
     public int $year_made;
     public float $zero_sixty;
     public float $price;
     public string $brand;
+    public int $id;
 
-    public function __construct(string $name = NULL, int $year_made = NULL, float $zero_sixty = NULL, float $price = NULL, string $brand = NULL)
+    public function __construct(?string $name = NULL, ?int $year_made = NULL, ?float $zero_sixty = NULL, ?float $price = NULL, ?string $brand = NULL)
     {
         $this->name = $name;
         $this->year_made = $year_made;
         $this->zero_sixty = $zero_sixty;
         $this->price = $price;
         $this->brand = $brand;
+
+        $method = new CarModel(make_db());
+        $this->id = $method->getIdFromName($this->name);
```

Comment:
Sorry this isn't quite what I had in mind! Using a model from an entity is violating single responsibility princple, all the entity should do is represent a row in the DB.

Instead of doing this, you should modify your model methods that select cars to also get the ID, and to then pass it into the Car whenever they get created.